kafka-elastic/src/main/java/com/baeldung/kafka/service/KafkaProducerService.java

```diff
@@ -0,0 +1,20 @@
+package com.baeldung.kafka.service;
+
+import com.baeldung.kafka.model.NotificationModel;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.stereotype.Service;
+
+@Service
+@Slf4j
+public class KafkaProducerService {
+
+    @Autowired
+    private KafkaTemplate<String, NotificationModel> kafkaTemplate;
+    public void sendMessage(NotificationModel notificationModel) {
```

Comment:
Please use the Baeldung formatter that should help with the expected newlines etc.

```suggestion
    private KafkaTemplate<String, NotificationModel> kafkaTemplate;

    public void sendMessage(NotificationModel notificationModel) {
```